src/teacup.coffee

```diff
@@ -174,6 +176,13 @@ class Teacup
     @raw contents
     @raw "</#{tagName}>"
 
+  scriptTag: (tagName, args...) ->
+    {attrs, contents} = @normalizeArgs args
+    @raw "<#{tagName}#{@renderAttrs attrs}>"
+    @raw @scriptescape contents
```

Comment:
Wouldn't you html escape this by default, and instead let the user explicitly pass a raw block into the script tag if it's safe content? It's tricky embedding a script tag with potentially dirty data. See: https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet for a quick reference. If you have an explicit example I can try breaking the template.